crypto/s2n_sequence.c

```diff
@@ -38,3 +40,17 @@ int s2n_increment_sequence_number(struct s2n_blob *sequence_number)
 
     return 0;
 }
+
+int s2n_convert_sequence_number(struct s2n_blob *sequence_number, uint64_t *output)
+{
+    notnull_check(sequence_number);
+    
+    int position = S2N_TLS_SEQUENCE_NUM_LEN - 1;
+    /* Each uint8_t can hold 2^8 values */
+    int power = 8;
+    for (int i = 0; i < sequence_number->size; i++) {
+        *output += (uint64_t) sequence_number->data[i] << (position * power);
+        position--;
+    }
+    return S2N_SUCCESS;
```

Comment:
A couple of things here:

1. this has an implicit assumption that `sequence_number->size < S2N_TLS_SEQUENCE_NUM_LEN`.  This should be guarded against.
2. we shouldn't be using `int` except for POSIX return types